packages/react-components/src/AccessibilityScenarios/Tooltip.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Button } from '@fluentui/react-button';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Tooltip } from '@fluentui/react-tooltip';
+
+import { Scenario } from './utils';
+
+export const ButtonsWithTooltipAccessibilityScenario: React.FunctionComponent = () => {
+  return (
+    <Scenario pageTitle="Buttons with tooltip">
+      <Tooltip content="Tooltip as a replacement label" triggerAriaAttribute="label">
```

Comment:
please add one more Tooltip without the `triggerAriaAttribute` to show the default behavior

Reply:
Okay, added.